plugin_tests/sources_test.py

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+#############################################################################
+#  Copyright Kitware Inc.
+#
+#  Licensed under the Apache License, Version 2.0 ( the "License" );
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#############################################################################
+
+import numpy
+import os
+import PIL.Image
+
+from girder import config
+from tests import base
+
+from . import common
+
+
+# boiler plate to start and stop the server
+
+os.environ['GIRDER_PORT'] = os.environ.get('GIRDER_TEST_PORT', '20200')
+config.loadConfig()  # Must reload config to pickup correct port
+
+
+def setUpModule():
+    base.enabledPlugins.append('large_image')
+    base.startServer(False)
+
+
+def tearDownModule():
+    base.stopServer()
+
+
+class LargeImageSourcesTest(common.LargeImageCommonTest):
```

Comment:
Need unit tests for `getRegion`? It is probably one of the functions that will be used quite a lot apart from `tileIterator`